src/corelibs/U2View/src/ov_msa/MultilineScrollController.h

```diff
@@ -138,11 +136,14 @@ public slots:
     int savedFirstVisibleMaRow;
     int savedFirstVisibleMaRowOffset;
 
+    bool checkBoundary();
+
     // TODO:ichebyki
     // Deprecated, will be removed
-    void centerBase(int baseNumber);
-    void centerPoint(const QPoint& maPoint);
-    void setCenterVisibleBase(int firstVisibleBase);
+    void _centerBase(int baseNumber);
+    void _centerPoint(const QPoint& maPoint);
+    void _setCenterVisibleBase(int firstVisibleBase);
+    void _centerViewRow(QPoint maPoint);
```

Comment:
Use const T& for all non-primitive types

Reply:
Fixed

Reply:
all these methods was removed